FF-Knoten der Telgter Kerngruppe

dg1rue opened the pull request:
55 Nodes hinzugefügt

dg1rue committed:
Update config.json

Nodes der Telgter FF-Gruppe hinzugefügt

dg1rue committed:
Merge pull request #1 from dg1rue/patch-1

Update config.json

MPW1412 commented:
https://forum.freifunk-muensterland.de/t/nodeliste-fuer-advanced-node-stats-in-grafana/3544/16?u=mpw

dg1rue committed:
Update config.json

Nodes von Domäne 07 hinzugefügt, die unter Kontrolle von "Kontakt@Freifunk-Telgte.net" sind.

dg1rue committed:
Merge branch 'glass_link' into glass_link

dg1rue commented:
Nur Nodes aufgenommen, die unter alleiniger Kontrolle von "Kontakt@Freifunk-Telgte.net" sind.

MPW1412 commented:
Sieht gut aus:

```
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "14cc20315716",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327ee43ba",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2226f88",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "e8de27fcfcb0",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c46e1ff4c22a",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c46e1f2e3376",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c222d804",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2ed2e16",
      "contact": "kontakt@freifunk-telgte.net"
    "node_id": "10feedf40870",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c4e984f08f5c",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c46e1f2d2c7c",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c4e984f0a738",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "14cc2063ae68",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "8416f97b7a7e",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "6466b3de6610",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "14cc20b9a9a4",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327ee39a2",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "ec086b837808",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2c28120",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "14cc20b9a9be",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "98ded09239f2",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327ee4370",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2ed1392",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327cee562",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327ee42f8",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "a42bb0b15cb8",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "a42bb0c2d0d4",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "44d9e7c71014",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "f4f26d4a30f4",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "44d9e77bbdf8",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2ed56c2",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327cee520",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c46e1fb656be",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "c46e1fb65578",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "98ded03c24ba",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c222705c",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "60e327ee43a6",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2cbb298",
      "contact": "Kontakt@Freifunk-Telgte.net"
    "node_id": "30b5c2ed13a6",
```
